File: Лабораторная №0/pascal/10.pas
var a,b,c, per: real;
begin
  write('Введите число: ');
  read(a);
  write('Введите число: ');
  read(b);
  write('Введите число: ');
  read(c);
  per := (a+b+c) / 2;
  writeln(sqrt(per*(per - a)*(per - b)*(per -c)));
end.

File: Лабораторная №0/python/10.py
from math import sqrt

a = float(input('Введите первую сторону треугольника: '))
b = float(input('Введите вторую сторону треугольника: '))
c = float(input('Введите третью сторону треугольника: '))
per = (a+b+c)/2
result = sqrt(per*(per -a)*(per-b)*(per-c))
print(result)